src/participant/prompts/exportToPlayground.ts

```diff
@@ -0,0 +1,32 @@
+import * as vscode from 'vscode';
+
+export class EportToPlaygroundPrompt {
+  static getAssistantPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert.
+Your task is to help the user build MongoDB queries and aggregation pipelines that perform their task.
+You achieve this by converting user's code written in any proggramming language to the MongoDB Shell syntax.
+Take a user promt as an input string and translate it to the MongoDB Shell language.
+Keep your response concise.
+You should suggest queries that are performant and correct.
+Respond with markdown, suggest code in a Markdown code block that begins with \`\`\`javascript and ends with \`\`\`.
```

Comment:
Have we tried to see how it performs if we get rid of the code block rules here and expect the response to already be code?

Reply:
It returns it as \`\`\`mongodb markdown, so to make it consistent with other places I found it cleaner to also specify it \`\`\`javascript in the prompt.